www/csp-incl.js

```diff
@@ -49,7 +49,13 @@ if (!window._disableCSP) {
     switch (PLAT) {
         case 'android':
         case 'ios':
-            cspMetaContent = 'default-src \'self\' https://ssl.gstatic.com/accessibility/javascript/android/; connect-src \'self\' http://cordova-filetransfer.jitsu.com;frame-src \'self\' data: gap:; img-src \'self\' data:; style-src \'self\' \'unsafe-inline\'';
+        case 'windows8':
+            cspMetaContent = 'default-src \'self\' https://ssl.gstatic.com/accessibility/javascript/android/;' +
+                            ' connect-src \'self\' http://cordova-filetransfer.jitsu.com;' +
+                            ' media-src \'self\' http://cordova.apache.org/downloads/;' +
+                            ' frame-src \'self\' data: gap:;' +
+                            ' img-src \'self\' data:;' +
+                            ' style-src \'self\' \'unsafe-inline\'';
```

Comment:
Even though the fact that this fixes things on Windows is actually really valuable, the formatting is the best thing about this change. Small nit: might be better to use double-quotes for the strings so you won't have to escape as much.